src/ClaimService.Mappers/Patch/PatchDbClaimMapper.cs

```diff
@@ -8,20 +8,5 @@ namespace LT.DigitalOffice.ClaimService.Mappers.Patch;
 
 public class PatchDbClaimMapper : IPatchDbClaimMapper
```

Comment:
Do you nedd this class?

Reply:
I can't see it. There is even no mappers project